src/bin/dual-iir.rs

```diff
@@ -69,7 +68,7 @@ const APP: () = {
     #[init(spawn=[telemetry, settings_update])]
     fn init(c: init::Context) -> init::LateResources {
         // Configure the microcontroller
-        let (mut stabilizer, _pounder) = hardware::setup(c.core, c.device);
+        let (mut stabilizer, _pounder) = setup(c.core, c.device);
```

Comment:
Honestly, I think `hardware::setup` is more descriptive than just `setup()`, but that's a personal preference. I just think the `hardware` portion makes it clear what type of setup is happening.

Reply:
Maybe but why is function called `setup()` while the module (made up to 90% by `setup()`) is called `configuration`? Shouldn't that be `configure()` then?

Reply:
I'm fine with naming either both `setup.rs` or `configuration.rs`, but I still think it would be useful to keep the `hardware` prefix here personally - could be convinced otherwise if we renamed to something like `hardware_setup(c.core, c.device)`. Anything else just feels far too generic to me.

Reply:
changed to `stabilizer::hardware::setup::setup()`.